templates/repo/search.tmpl

```diff
@@ -44,20 +44,7 @@
 								<a role="button" class="ui basic tiny button" rel="nofollow" href="{{$.SourcePath}}/src/commit/{{PathEscape $result.CommitID}}/{{PathEscapeSegments .Filename}}">{{ctx.Locale.Tr "repo.diff.view_file"}}</a>
 							</h4>
 							<div class="ui attached table segment">
-								<div class="file-body file-code code-view">
-									<table>
-										<tbody>
-											<tr>
-												<td class="lines-num">
-													{{range .LineNumbers}}
-														<a href="{{$.SourcePath}}/src/commit/{{PathEscape $result.CommitID}}/{{PathEscapeSegments $result.Filename}}#L{{.}}"><span>{{.}}</span></a>
-													{{end}}
-												</td>
-												<td class="lines-code chroma"><code class="code-inner">{{.FormattedLines}}</code></td>
-											</tr>
-										</tbody>
-									</table>
-								</div>
+								{{template "shared/searchfile" dict "repolink" $.SourcePath "result" .}}
```

Comment:
Could it follow most other `{{template}}` styles?


```suggestion
								{{template "shared/searchfile" dict "RepoLink" $.SourcePath "SearchResult" .}}
```

Reply:
Do we have a clearly rule for this? I also confused which one to use. At last, I followed the format in L49.

![image](https://github.com/go-gitea/gitea/assets/18380374/ddb19670-717e-433d-81d2-32b6f4d6458e)

Reply:
> Do we have a clearly rule for this? I also confused which one to use. At last, I followed the format in L49.

"root" is special ........ it means global.